src/main/java/org/jenkinsci/plugins/workflow/cps/GroovyClassLoaderWhitelist.java

```diff
@@ -31,17 +35,17 @@
      */
     private final Whitelist delegate;
 
-    GroovyClassLoaderWhitelist(GroovyClassLoader scriptLoader, Whitelist delegate) {
-        this.scriptLoader = scriptLoader;
+    GroovyClassLoaderWhitelist(Whitelist delegate, GroovyClassLoader... scriptLoaders) {
+        this.scriptLoaders = Arrays.<ClassLoader>asList(scriptLoaders);
```

Comment:
`IdentityHashMap<ClassLoader,Void>` obscures the code and I have never seen any funny `ClassLoader.equals()` implementation, so I don't think it's a problem.